### ISSUE TITLE
OSX High Sierra

### ISSUE THREAD

**Issue by julianpoy** [AndrewDryga/vagrant-box-osx#64]
High Sierra version of the box would be nice now that it's released.

**Comment by dotdoom** [comment]
Agreed, High Sierra brings Xcode 9.x which is the version officially supported by e.g. https://github.com/flutter/flutter.

Updating via `softwareupdate -a -i --verbose` gets stuck after downloading and unpacking files. Updating via UI also gets stuck a few minutes after reboot.

**Comment by rafaelrpinto** [comment]
I've been able to upgrade the box to High Sierra. Usually when you upgrade after the restart nothing happens. You need to perform a workaround described here: 
https://raimue.blog/2017/06/09/upgrading-a-vm-from-macos-10-12-sierra-to-macos-10-13-high-sierra-in-virtualbox/

This workaround have been working with all updates so far. (virtualbox 5.2)

**Comment by chaospixel** [comment]
monsenso appears to have one https://app.vagrantup.com/monsenso/boxes/macos-10.13; although I couldn't get it to start. That's because I can't get a VirtualBox more recent than 5.1.18 to actually work on my MacBook Pro though...

**Comment by omkarnathsingh** [comment]
How to get latest macOS High Sierra? Will this repo be updated?

**Comment by GregJPreece** [comment]
Can confirm that the Vagrant box linked by @chaospixel works correctly, though it has the usual performance issues of OSX in a VM. It **requires** at least VirtualBox 5.2 (5.2.2 and above for the resolution fix below), with the extensions pack installed. 

I can't find a Github/FOSS repo for the box, so I can't submit this there, but if you're using it and have problems with screen resolution, you can override them in the Vagrantfile:

```
config.vm.provider "virtualbox" do |vb|

  ... Existing config here ...

  # Set the screen resolution inside the machine
  vb.customize ["setextradata", :id, "VBoxInternal2/EfiGraphicsResolution", "1920x1080"]

end
```

Replace 1920x1080 with whatever screen resolution you desire, obviously.
